src/runtime-rs/crates/runtimes/common/src/container_manager.rs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+// Copyright (c) 2019-2022 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use anyhow::Result;
+use async_trait::async_trait;
+
+use crate::types::{
+    ContainerConfig, ExecProcessRequest, KillRequest, ProcessExitStatus, ProcessID,
+    ProcessStateInfo, ResizePtyRequest, ShimPid, ShutdownRequest, StatsInfo, UpdateRequest,
+};
+
+#[async_trait]
+pub trait ContainerManager: Send + Sync {
+    // container lifecycle
+    async fn create_container(&self, config: ContainerConfig) -> Result<u32>;
+    async fn pause_container(&self, cid: &str) -> Result<()>;
+    async fn resume_container(&self, cid: &str) -> Result<()>;
+    async fn stats_container(&self, cid: &str) -> Result<StatsInfo>;
+    async fn update_container(&self, req: UpdateRequest) -> Result<()>;
+
+    // process lifecycle
+    async fn close_io_process(&self, process_id: &ProcessID) -> Result<()>;
```

Comment:
The naming convention here clashes with English grammar. Normally, it would read `close_process_io` or `close_io_for_process`.

Possible suggestions (one or the other):
1. Put the category first, i.e. `container_create`, `process_close_io`, etc.
2. Create separate traits for the container lifecycle, process lifecyclce and utility (that might prove handy later for confidential containers where process lifecycle will more naturally talk to the host, whereas a lot of the container lifecycle will be shifted to the agent)

Reply:
Thanks for your review, Considering that it is all container-related processing, I think it's unnecessary to separate by traits and try to rename `close_io_process` to `close_process_io`.